versioning/revisions.rst

```diff
@@ -76,6 +76,58 @@ Take into account that it changes the default Conan behavior. e.g:
     - If you generate and upload N binary packages for a recipe with a given revision, then if you modify the recipe, and thus the recipe
       revision, you need to build and upload N new binaries matching that new recipe revision.
 
+
+GIT and Line Endings on Windows
+-------------------------------
+
+There is one very common problem that users encounter after enabling revisions which all users should be aware of. 
+This issue occurs when all of the following conditions are true:
+
+ - Using GIT to retrieve sources with default settings
+ - Using Continuous Integration services
+ - Building on multiple platforms
+ - One of the target platforms is Windows
+
+When cloning a repository on a Windows machine, the GIT client will replace all of the line endings on all of the 
+files with @CRLF character. The default line ending outside of Windows is @LF.  
+
+**Problem**
+
+As a result, when Conan does export of an otherwise identical GIT repository/commit on a Windows and a Non-Windows build machine,
```

Comment:
I think this section (and the docs in general) can be highly reduced and be as concise as possible and to the point (without losing information). Same as the workaround section. Something like:

Problem
Git will (by default) checkout files in Windows systems using CRLF line endings, effectively producing different files. As files are different, the Conan revisions will be different from the revisions computed in other platforms as Linux, resulting in missing the respective binaries in the other revision.

Solution
It is necessary to instruct Git to do the checkout with the same line endings. This can be done with a ``.gitattributes`` file, .....


At this point is more important to highlight with a ``warning`` or some other decorator about this potential problem so users don't skip reading this.

Reply:
Resolved in https://github.com/conan-io/docs/pull/2028